### ISSUE TITLE
Show "completed programs" section to applicant when browsing list of programs

### ISSUE THREAD

**Issue by kendrickt** [civiform/civiform#1393]
**Is your feature request related to a problem? Please describe.**
Applicant sees all programs available, regardless of whether they've finished applying to them or not. This can be confusing, and clutters the list.

**Describe the solution you'd like**
The applicant's program index view should have three distinct sections for programs. Right now there are just two: 
1. in-progress programs
2. everything else.

The "everything else" section should be split into two sections: new and completed programs.

**Done when**
Applicants should be able to quickly identify between:
1. programs they are in the middle of applying for
2. programs they haven't started applying for
3. programs they've finished applying for

**Comment by cassandralfernandes** [comment]
@clouser2 you worked on this through issue #2573 right? Any objections to closing this issue as a duplicate?

**Comment by clouser2** [comment]
Correct that's part of #2573. The implementation for that has been ready for a bit, we're just waiting on translated content for the new section headings before merging it in (https://github.com/civiform/civiform/pull/2695).

**Comment by clouser2** [comment]
Duplicate of #2573
